Fabric.Authorization.Persistence.SqlServer/Extensions/ModelBuilderExtensions.cs

```diff
@@ -130,6 +135,10 @@ public static void ConfigureRole(this ModelBuilder modelBuilder)
             modelBuilder.Entity<Role>(entity =>
             {
                 entity.ToTable("Roles");
+
+                entity.Property(e => e.Id)
+                    .UseSqlServerIdentityColumn();
```

Comment:
Should this have IsUnique?

Reply:
I have the unique property on the index setup